src/learn/qml/MuseScore/Learn/internal/ClassesPage.qml

```diff
@@ -170,7 +170,7 @@ FocusScope {
 
             StyledTextLabel {
                 height: implicitHeight
-                width: parent.width
+                width: 520
```

Comment:
authorInfo limits the text size(see anchors.right).
If you need to correct the text size, then only there.